### PR TITLE
F-12: UDPMockserver

### LINKED ISSUE #12: Write MockUDPGenerator
Write a generator that would generate mock UDP data so that testing can be done without having the game open.

This should be done before #5

### PR DESCRIPTION
**Pull request by adamspriggs**
Fixes #12

### CODE REVIEW

**Comment by adamspriggs** [comment]
@ryanabaker If you want to look at this one (My VS updated to version 17). Little sus about the merge conflicts and just adding the project files as I did. Should work theoretically.

**Review by ryanabaker — APPROVED** [review]

**Comment by ryanabaker** [comment]
Going to require PacketSessionHistoryData as I just added it, I can do later don't matter

File: F1T.sln [hunk 1]
```diff
@@ -4,7 +4,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 VisualStudioVersion = 17.0.32112.339
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "F1T", "F1T\F1T.csproj", "{95FACA9F-4C85-40AB-A672-B5E508E37161}"
-EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "F1TMock", "F1TMock\F1TMock.csproj", "{981B8F8C-2E07-4A4B-877B-EBD1738F069D}"
```
Comment (adamspriggs) [inline F1T.sln]: @ryanabaker I have to test this later. These two project lines have the same ID at the start

Comment (ryanabaker) [reply F1T.sln]: im confusion, but yes

Comment (ryanabaker) [reply F1T.sln]: VS auto changed it when opening

see https://github.com/F1Telemetry/F1T/pull/35/commits/60441114b73b524f38f4e84e005163a0f31460ed

**Pull request merged**